src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -1,35 +1,57 @@
 package seedu.address.logic.parser;
 
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.ParserUtil.parseWhitespaceSeparatedStrings;
 
-import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.DeleteByIndexCommand;
+import seedu.address.logic.commands.DeleteByTagCommand;
 import seedu.address.logic.commands.DeleteCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
  * Parses input arguments and creates a new DeleteCommand object
  */
 public class DeleteCommandParser implements Parser<DeleteCommand> {
-
+    /**
+     * Utility function to check that the input arguments is not empty.
+     * Throws a parse exception if it is empty.
+     */
+    private void checkArgsNotEmpty(String args) throws ParseException {
```

Comment:
Just like to add that this is a good method which could be expanded as a helper-method for global usage. Probably exist as a method in `OptionBearingArgument` with a parameter to check which command is calling it (for ParseException message). I see that it's in `DeleteCommandParser` as well as `FindCommandParser`. 

I wanted to use it for my `ThemeCommandParser` too. :P